### DISCUSSION TITLE
Can I create a Transaction History report which aggregates accounts?

### DISCUSSION THREAD

**Discussion by Terry-JF** [firefly-iii/firefly-iii#9083] [Firefly III]
I have a lot of transfer transactions that move funds between accounts. I can use the Transaction History report to see all transactions for the previous 7 days (across all accounts) but these are listed in seperate account groups according to the Account each transaction is linked to. This means a lot of scrolling to follow funds moving around, plus each transfer is listed twice, in both the source and destination account. Is there any way to get this report to simply list these transactions in a single date ordered list and with transfers showing as a single entry?

I could export the transactions into a spreadsheet, remove the duplicate transfers and then sort them, but doing this within FF3 would be nice.

Thanks,
Terry

**Comment by JC5** [comment]
Mmm, I'm afraid not. You could build something with the API though?
